cli/commands/site/create.ts

```diff
@@ -275,6 +263,42 @@ function readBlueprint( blueprintPath: string ) {
 	}
 }
 
+export async function coerceBlueprint( value: string ) {
+	let blueprintJson: unknown;
+
+	if ( value ) {
+		if ( /^https?:\/\//.test( value ) ) {
+			blueprintJson = await fetchBlueprint( value );
+		} else {
+			blueprintJson = readBlueprint( value );
+		}
+	}
+
+	return blueprintJson;
+}
```

Comment:
```suggestion
export async function coerceBlueprint( value: string ) {
	if ( value ) {
		if ( /^https?:\/\//.test( value ) ) {
			return await fetchBlueprint( value );
		} else {
			return readBlueprint( value );
		}
	}
}
```
Nit. Now that this lives in a standalone function, we don't really need to assign to the `blueprintJson` variable.

Reply:
👍
Also, we don't need `if ( value ) {`, since the coerce function runs only if the argument is provided. One more small benefit of using this approach.